vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -53,16 +61,16 @@ ent bundle run conference-table
 1. In your browser, go to <http://localhost:3000>
 2. Access the conference-table MFE with the default credentials of `username: admin`, `password: admin` 
 
-> Note: Once authenticated, the message "No conferences are available" is generated. If you check your browser console, you should see a `403 (Forbidden)` error for the request made to `localhost:8080/services/conference/api/conferences`. This is expected because the `admin` user has not yet been granted the new role. 
+**Note**: Once authenticated, the message "No conferences are available" is generated. If you check your browser console, you should see a `403 (Forbidden)` error for the request made to `localhost:8080/services/conference/api/conferences`. This is expected because the `admin` user does not yet have the necessary role.  
 
 ### Step 4: Login to Keycloak
 
 1. Go to <http://localhost:9080> 
-2. Login using the default credentials of `username: admin`, `password: admin`
+2. Log in using the default credentials of `username: admin`, `password: admin`
 
 ### Step 5: Create the `conference-user` and `conference-admin` roles 
 
-1. Go to `Clients` → `internal` → `Roles`
+1. Go to `Clients` → `internal` → `Roles` tab
```

Comment:
Feels odd to start adding 'tab' everywhere. Just my 2 cents. Feels like we'd have to specify the type of the target (button, tab, section, etc.) if we did that everywhere and I think it's fairly intuitive without it.

Reply:
I did that because there's a left menu option for Roles. But it is more intuitive that it would refer to the tab Roles.